pycbc/workflow/minifollowups.py

```diff
@@ -652,3 +655,10 @@ def create_noop_node():
     exe.add_pfn(pfn)
     node = wdax.Node(exe)
     return node
+
+def add_wiki_row(outfile, cols):
```

Comment:
I think for now, it's better to put it in the script. If we decide we need it, it can be moved to the library.